src/org/omegat/gui/main/ProjectUICommands.java

```diff
@@ -550,48 +540,21 @@ c. remote mapping, no local mapping(s)
                             }
                         }
                     }
-
-                    final ProjectProperties propsP = props;
-                    final File rewriteOnSuccessFinal = rewriteOnSuccess;
-                    final boolean needToSavePropertiesFinal = needToSaveProperties;
-                    // We have to extract the processing related to needToSaveProperties{,Final}
-                    // out of  the code block of  Core.executeExclusively()
-                    // and place it here, so that we can change the value of needToSaveProperties
-                    // in our update scheme for teamwork setting.
-                    // Sanity check of possibly modified new project property.
-                    final boolean succeeded = ProjectFactory.loadProject(propsP, true);
-                    if (rewriteOnSuccessFinal != null) { // teamwork case.
-                        if (succeeded) {    // remote content with minor mods seems sane.
-                            if (needToSavePropertiesFinal) {
-                                Log.logWarningRB("REMOTE_PROJECT_CONTENT_WAS_MODIFIED_TO_ADD_MAPPING_SO_NOT_USED");
-                                // Do not save the content of the remote file because we missed mapping.
-                                // So in our new scheme of backup and updating, we don't need to save the file
-                                // after this code block any more.
-                                needToSaveProperties = false;
-                                Files.deleteIfExists(rewriteOnSuccessFinal.toPath()); // remove "omegat.project.NEW"
-                            }
+                    // loading modified new project property
+                    final boolean succeeded = ProjectFactory.loadProject(props, true);
```

Comment:
Previous commit bcee3c8 tell us call `ProjectFactory.loadProject` should protect with `Core.executeExclusively`

https://github.com/omegat-org/omegat/blob/bcee3c8cf6ad9cef5789237ee60c353f10e994cc/src/org/omegat/gui/main/ProjectUICommands.java#L526-L542

Reply:
Thank you for tagging them.